inputs/worker_inner:2.json

```diff
@@ -0,0 +1,174 @@
+{
+  "client_version": "{{VERSION}}",
+  "prebuild_plugins": [
+    {
+      "name": "add_filesystem",
+      "args": {
+        "koji_hub": "{{KOJI_HUB}}",
+        "repos": [],
+        "from_task_id": "{{FROM_TASK_ID}}"
+      }
+    },
+    {
+      "name": "pull_base_image",
+      "args": {
+        "parent_registry": "{{REGISTRY_URI}}",
+        "parent_registry_insecure": true
+      }
+    },
+    {
+      "name": "bump_release",
```

Comment:
This plugin has been removed from `worker_inner:1.json`, let's remove it here too.